library/src/permutation/hiptensor_permutation.cpp

```diff
@@ -36,5 +38,5 @@ hiptensorStatus_t hiptensorPermutation(const hiptensorHandle_t*           handle
                                        const hipDataType                  typeScalar,
                                        const hipStream_t                  stream)
 {
-    return HIPTENSOR_STATUS_SUCCESS;
+    return hiptensor::details::permuteByCpu(alpha, A, descA, modeA, B, descB, modeB, typeScalar);
```

Comment:
This function will deduce DataType template argument as 'void'. Is this intended?

Reply:
See comment below on the permuteByCpu function